portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path

```diff
@@ -61,6 +61,54 @@
 	<td></td>
 </tr>
 
+<!-- AB Test -->
+
+<tr>
+	<td>AB_GOAL_SELECTION</td>
+	<td>//select[contains(@class,'form-control mt-1')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>AB_TEST_ICON</td>
+	<td>//*[contains(@class, 'lexicon-icon lexicon-icon-test')]</td>
+	<td></td>
+</tr>
```

Comment:
Hey @joshitagaki, I know that using macros and path from other teams is dangerous if they make any changes that could affect our tests, but I think that in this case the icon would be a simple path and should not be modified. I think we can use the tango team path to not create another one that does the same thing. What do you think?

`ABTest#AB_TEST_BUTTON`

Reply:
Hmm yea I was debating whether to just use the tango teams macros or not since the steps to create an ab test is functionally the same. I'll make a thread about this to get others opinion but personally I'd like to use their functionality as it would be redundant to have our own and it more important in our test to make sure the test is showing in AC

Reply:
Exactly, I was thinking the same thing. The important thing for us is to verify that the test appears in the AC. I think you could analyze their macros and check if they're following your steps, if the macros are doing more complex steps than yours, then maybe we'd better create our own macro

Reply:
I think that anything in the AC product itself or in the DXP connection screen (aka, anything our development team maintains) should be using our own paths.

Anything that is developed by Tango team (parts of A/B test inside of DXP) should be maintained by them.

HOWEVER, we have not made this agreement with their team yet, so I will reach out to start a conversation with them about how we want to divide things in the future.  Probably we will have to refactor some of their tests (and maybe some of our tests) to use the right macros and paths.

Reply:
I've created a thread to begin this conversation: https://liferay.slack.com/archives/C03GHESJRPT/p1660781376589779